### ISSUE TITLE
Search feature in GGC

### ISSUE THREAD

**Issue by Tam51** [TheGameCreators/GameGuruRepo#3876]
Hi There

I think it would be nice to have a search feature in GG Classic. It would make it easier to find models in GGC. Any chance of adding this to GGC like we have in GGmax. I think it would be cool and faster to find the right model instead of having to look through everyone to find what you're looking for.

THKs

**Comment by Captain-Perl** [comment]
Here is a little **workaround** - not as good as an internal search option, but simple and free.
1. open a Command Window (Shell)
2. go to the Game Guru\Files\entitybank folder
3. Enter this: `dir /s >../entities.txt`
4. Now in your Files folder, you have a new text file with all your entities
5. Open the file with an editor of your choice (e.g. Notepad++) - and you can search in it! When you have found an object, scroll a little up to see the folder where it is located. Then you know where it is in GG.
6. Exit the shell. (You have to create the file only once.)
7. Of course, when you have new entities then you have to repeat the process for an index update

This trick saved me much time looking for certain entities.

**Comment by Teabone3** [comment]
I think its fair to say we will not be getting this feature or addition to GGC. Unless it was open-sourced someday in the future.
